src/kioskboard.js

```diff
@@ -760,10 +760,15 @@
               var scrollBehavior = opt.cssAnimations === true ? 'smooth' : 'auto';
               var scrollDelay = opt.cssAnimations === true && typeof opt.cssAnimationsDuration === 'number' ? opt.cssAnimationsDuration : 0;
               var userAgent = navigator.userAgent.toLocaleLowerCase('en');
+              var isEdgeWebView = userAgent.indexOf('edge') > -1 && userAgent.indexOf('webview') > -1;
```

Comment:
Hi @surexxx ,

First of all thanks for your contributions.

Could you please make these a variable:

```js
var isBrowserEdgeLegacy = userAgent.indexOf('edge') > -1;
var isBrowserInternetExplorer = userAgent.indexOf('.net4') > -1;
var isBrowserEdgeWebView = isBrowserEdgeLegacy && userAgent.indexOf('webview') > -1;

if((!isBrowserEdgeLegacy || isBrowserEdgeWebView) && !isBrowserInternetExplorer) {
  ...
}
```

Thanks,
Furkan.